docs/en/interfaces/formats.md

```diff
@@ -2445,6 +2446,26 @@ Result:
 └──────────────┘
 ```
 
+## Npy {#data-format-npy}
+
+This function is designed to load a NumPy array from a .npy file into ClickHouse. The NumPy file format is a binary format used for efficiently storing arrays of numerical data. It stores all the top-level dimension objects as a separate column.
```

Comment:
Need a note that we trean top-level array as an array of rows with single column.
Also we should write what numpy data types we support.